spacy_llm/tasks/templates/ner.v3.jinja

```diff
@@ -0,0 +1,19 @@
+{{ description }}
+{# whitespace #}
+{# whitespace #}
+Q: Given the paragraph below, identify a list of possible entities, and for each entry explain why it is or is not an entity:
+{# whitespace #}
+{# whitespace #}
+{%- for example in examples -%}
+Paragraph: {{ example.text }}
+Answer:
+{# whitespace #}
+{%- for span in example.entities -%}
+{{ loop.index }}. {{ span.to_str() }}
+{# whitespace #}
```

Comment:
This confuses me. The prompt says to do a binary yes/no classification, but here we only use positive examples, correct?